cloud-aws/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/component/AwsLaunchTest.java

```diff
@@ -176,10 +199,18 @@ public void launchStack() throws Exception {
         setupDescribeSubnetResponse();
         setupDescribePrefixListsResponse();
 
+        setupCreateFileSystem();
+        setupDescribeFileSystems();
+        setupDescribeMountTargets();
+        setupDeleteMountTarget();
+        setupDeleteFileSystem();
+
```

Comment:
Let's create a new test instead of changing the old one. Leave the old one as-is for regression testing.